candy-kata/README.md

```diff
@@ -0,0 +1,34 @@
+<!--
+  ~ Copyright (c) 2020 The Bank of New York Mellon.
```

Comment:
This copyright date should be updated to 2021.

Reply:
Updated